packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+| roleId      | `id` of a Role                                                 | URL          | String     | TRUE     |
+| userId      | `id` of a User                                                 | URL          | String     | TRUE     |
+
+Treat the page cursor as an opaque value: obtain it through the next link relation. See [Pagination](/docs/reference/api-overview/#pagination).
+
+###### Response parameters
+
+Array of [Groups](/docs/reference/api/groups/)
+
+If the Role isn't scoped to specific Group targets, an empty array `[]` is returned.
+
+###### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/users/00u6fud33CXDPBXULRNG/roles/KVJUKUS7IFCE2SKO/targets/groups"
+```
+
+###### Response example
+
+
+```json
+[
+  {
+    "id": "00g1emaKYZTWRYYRRTSK",
+    "objectClass": [
+      "okta:user_group"
+    ],
+    "profile": {
+      "name": "West Coast Users",
+      "description": "All Users West of The Rockies"
+    },
+    "_links": {
+      "logo": [
+        {
+          "href": "https://${yourOktaDomain}/img/logos/groups/okta-medium.png",
+          "name": "medium",
+          "type": "image/png"
+        },
+        {
+          "href": "https://${yourOktaDomain}/img/logos/groups/okta-large.png",
+          "name": "large",
+          "type": "image/png"
+        }
+      ],
+      "users": {
+        "href": "https://${yourOktaDomain}/api/v1/groups/00g1emaKYZTWRYYRRTSK/users"
+      },
+      "apps": {
+        "href": "https://${yourOktaDomain}/api/v1/groups/00g1emaKYZTWRYYRRTSK/apps"
+      }
+    }
+  }
+]
+```
+
+##### List group targets for a group administrator role given to a group
+
+<ApiOperation method="get" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/groups" />
+
+Lists all Group targets for a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a Group
+
+###### Request parameters
+
+
+| Parameter       | Description                                                    | Param Type   | DataType   | Required |
+| :-------------- | :------------------------------------------------------------- | :----------- | :--------- | :------- |
+| after           | Specifies the pagination cursor for the next page of targets   | Query        | String     | FALSE    |
+| groupId         | `id` of a Group                                                | URL          | String     | TRUE     |
+| limit           | Specifies the number of results for a page (default is 20)     | Query        | Number     | FALSE    |
+| roleId          | `id` of a Role                                                 | URL          | String     | TRUE     |
+
+Treat the page cursor as an opaque value: obtain it through the next link relation. See [Pagination](/docs/reference/api-overview/#pagination).
```

Comment:
Treat the page cursor as an opaque value: obtain it through > Treat the page cursor as an opaque value. You can obtain it through